fabric-command-api-v2/src/client/java/net/fabricmc/fabric/api/client/command/v2/ClientCommandManager.java

```diff
@@ -75,6 +79,29 @@ private ClientCommandManager() {
 		return ClientCommandInternals.getActiveDispatcher();
 	}
 
+	/**
+	 * Refresh the command completions. This is helpful when a condition as defined using {@link LiteralArgumentBuilder#requires}
+	 * changes for a client command. The method uses the last received {@code minecraft:commands}
+	 * packet and calls its handler. This triggers the client command's condition to be reevaluated.
+	 *
+	 * <p>Will not do anything when not connected to a server (dedicated or integrated).</p>
```

Comment:
Not super keen on just silently failing with no way to know if it worked or not. I would throw an a mod can choose to catch it or not.